web-portal/pages/admin-venue-edit/index.vue

```diff
@@ -57,11 +63,50 @@
       }
     },
 
+    mounted: function () {
+      this.$store.dispatch('admin/LoadUnverifiedEvents', { idToken: getToken(this.$auth) })
+      this.$store.dispatch('admin/LoadCurrentEvents', { idToken: getToken(this.$auth) })
+      this.$store.dispatch('admin/LoadResourceEvents', { idToken: getToken(this.$auth) })
+
```

Comment:
Is this a copy-paste mistake? I don't think this page needs to fetch all this event data.

Reply:
this was existing code I believe not sure why it's showing in the pr as new, let me look at that

Reply:
oh nvm, yeah that must be copy/paste mess up, fixed